tdrs-backend/tdpservice/email/email.py

```diff
@@ -0,0 +1,88 @@
+"""Send emails."""
+
+from celery import shared_task
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.core.mail import EmailMultiAlternatives
+from django.conf import settings
+from django.template.loader import get_template
+from django.template import TemplateDoesNotExist
+from tdpservice.email.email_enums import EmailType
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+@shared_task
+def construct_email(email_type, context):
+    """Get email template."""
+    template_path = email_type.value + ".html"
+    try:
+        template = get_template(template_path)
+    except TemplateDoesNotExist:
+        logger.error(f'Template {template_path} does not exist')
+        return
```

Comment:
I sorta think this error handling is inappropriate. Rather then catching a logging it is seems like it might be better to let the traceback display in the console so it is easier to find. I had this happen and since None was returned the next function failed and I missed this log statement, which was above the traceback I was using to debug.